test.Dockerfile

```diff
@@ -2,7 +2,10 @@ FROM alpine:latest AS core
 
 RUN apk update
 RUN apk upgrade
-RUN apk add --update go gcc g++ vips vips-dev
+RUN apk add --update bash cmake git gcc g++ make vips vips-dev
+
+COPY --from=golang:1.20-alpine /usr/local/go/ /usr/local/go/
```

Comment:
how does this work?

Reply:
It seemed like the easiest solution among these: https://stackoverflow.com/questions/52056387/how-to-install-go-in-alpine-linux. I also like that we can lock the go version using the docker tag there.